Gemfile.local

```diff
@@ -0,0 +1,2 @@
+# replace google with your oauth gem here
+gem 'omniauth-google-oauth2'
```

Comment:
I would ignore `Gemfile.local` since this file is going to possibly be changed. In my case, I would change this to `gem 'omniauth-gitlab'`